cron_sentry/runner.py

```diff
@@ -152,22 +164,32 @@ def run(self):
             with TemporaryFile() as stderr:
                 try:
                     exit_status = call(self.command, stdout=stdout, stderr=stderr)
+                    last_lines_stdout = self._get_last_lines(stdout)
```

Comment:
any reason to move the `last_lines_stdout` and `last_lines_stderr` lines to inside the `try`, @rogersprint? The reason they were outside originally is that only `call()` should be raising `CommandNotFoundError`.